docs/release_docs/AHA-Prod_ReleaseFlow.md

```diff
@@ -0,0 +1,36 @@
+# AHA-Prod Workflow
+Explaination of AHA-PROD workflow.
+![AHA-PROD](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/AHA-PROD_Workflow.png?raw=true)
+
+## AHA-PROD-ci-cd
+
+This workflow allows you to manually trigger a GitHub Action with a input Github ``` Tag_name ```, without having to push or create a pull request
+
+### JOBS
+
+#### Github-ECR-Tag-Check
+This is how Github-ECR-Tag-Check job looks like
+
+![github-ecr-tag-checks](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/Github-ECR-Tag-Check.png?raw=true)
+
+* This job will check whether the input Github tag name is correct or not.
+* This job get the github release with the associated Github tag name with it and store the Github release ID.
+* Then it will check ECR image tag with the same as Github release ID.
+* This job uses [mukunku/tag-exists-action](https://github.com/marketplace/actions/tag-exists-action), [git-get-release-action](https://github.com/marketplace/actions/git-get-release-action).
+
+#### Deploy-ECS
+This is how Deploy-ECS job looks like
+
+![deploy-ecs](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/docker-build.png?raw=true) 
+
+* This job uses 'aha-PROD' environment and login to ECR using creds and pull the ECR image which was created in PROD-ci-cd with the github release ID Then this job will create new version of Amazon ECS task definition with new docker image then it will deploy Amazon ECS task definition using Duplo API.
```

Comment:
Environment names are case sensitive. The once we are using is `aha-prod` not `aha-PROD`